exercises/concept/ellens-alien-game/.docs/hints.md

```diff
@@ -0,0 +1,22 @@
+# Hints
+
+## 1. Create the Alien Class
+
+- The `constructor` does not have a return type and has the same name as the `class`.
+- Usually, the names of `classes` start with a capital letter.
```

Comment:
PascalCase for classes is a convention that I follow myself, too, but it's not the only convention. In fact, the standard library and [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-camel) use "snake_case" for types. Therefore I think that "usually" is a little bit too strong here.  
How about something like "A common convention for \`classes\` is using names that start with a capital letter."